### PR TITLE
Added concepts that were needed to model new IAI apartment.

### PR DESCRIPTION
**Pull request by daniel86**
I only added a few concepts, including comments.
Some axioms will follow in another pull request.

Not sure though why there is so much refactoring ongoing. I edited with Protege 5.5.0

### CODE REVIEW

File: owl/SOMA-HOME.owl [hunk 1]
```diff
@@ -12,59 +12,43 @@ Ontology(<http://www.ease-crc.org/ont/SOMA-HOME.owl>
...
-Declaration(Class(:TableSpoon))
-Declaration(Class(:TeaSpoon))
-Declaration(Class(:TemperingByCooling))
-Declaration(Class(:TemperingByHeating))
-Declaration(Class(:TrashContainer))
-Declaration(Class(:WaterBottle))
-Declaration(Class(:WaterGlass))
-Declaration(Class(:WineBottle))
-Declaration(Class(:WineGlass))
+Declaration(Class(<http://www.ease-crc.org/ont/SOMA-HOME.owl#Armchair>))
```
Comment (mpomarlan) [inline owl/SOMA-HOME.owl:15]: I'm a bit confused by the prefixes situation.

New concepts appear here with SOMA-HOME.owl in the prefix. At the same time, something like NutellaJar (seems too specific to be in SOMA proper) uses SOMA.owl as prefix. This is not an issue with this PR as such, rather it is something to keep in mind, perhaps by opening a github issue.

**Review by mpomarlan — APPROVED** [review]

**Comment by mpomarlan** [comment]
What's up with the SOMA Vocabulary step?

**Comment by daniel86** [comment]
> What's up with the SOMA Vocabulary step?

to me it looks like the CI step cannot handle the new format. But I haven't looked deeper into it

File: owl/SOMA-HOME.owl (hunk 1 above)
Comment (daniel86) [reply owl/SOMA-HOME.owl:15]: There is a related issue here: https://github.com/ease-crc/soma/issues/288

**Comment by daniel86** [comment]
>

**Comment by daniel86** [comment]
> > What's up with the SOMA Vocabulary step?
> 
> to me it looks like the CI step cannot handle the new format. But I haven't looked deeper into it.
> 
> seems owl2ready is used: https://github.com/ease-crc/soma/blob/master/docs/vocab/owl_reader.py#L5

"Owlready2 currently reads the following file format: RDF/XML, OWL/XML, NTriples. The file format is automatically detected." they write here https://owlready2.readthedocs.io/en/latest/onto.html

**Comment by daniel86** [comment]
> What's up with the SOMA Vocabulary step?

yeah it looks like owl2ready cannot handle functional syntax, which is the format of the file passed to it here: https://github.com/ease-crc/soma/blob/master/docs/vocab/SOMA-vocabulary.tex#L21
However, it looks like the wrong file is passed, i.e., not the bundled version. So I guess a fix would be to instead supply the bundled version in XML format to owl2ready.

**Pull request merged**